internal/namespaces/baremetal/v1alpha1/custom_server_create.go

```diff
@@ -0,0 +1,146 @@
+package baremetal
+
+import (
+	"context"
+	"log"
+	"reflect"
+
+	"github.com/scaleway/scaleway-cli/internal/core"
+	baremetal "github.com/scaleway/scaleway-sdk-go/api/baremetal/v1alpha1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+type createServerRequest struct {
+	Zone scw.Zone `json:"-"`
+	// OfferID offer ID of the new server
+	//OfferID string `json:"offer_id"`
+	// OrganizationID organization ID with which the server will be created
+	OrganizationID string `json:"organization_id"`
+	// Name name of the server (≠hostname)
+	Name string `json:"name"`
+	// Description description associated to the server, max 255 characters
+	Description string `json:"description"`
+	// Tags tags to associate to the server
+	Tags []string `json:"tags"`
+	// We want to run only with type
+	Type string
+}
+
+func serverCreateCommand() *core.Command {
+	return &core.Command{
+		Short:     `Create server`,
+		Long:      `Create a new server. Once the server is created, you probably want to install an OS.`,
+		Namespace: "baremetal",
+		Verb:      "create",
+		Resource:  "server",
+		ArgsType:  reflect.TypeOf(createServerRequest{}),
+		ArgSpecs: core.ArgSpecs{
+			core.ZoneArgSpec(scw.ZoneFrPar2),
+			core.OrganizationIDArgSpec(),
+			{
+				Name:    "type",
+				Short:   "Server commercial type",
+				Default: core.DefaultValueSetter("GP-BM1-M"),
+
+				EnumValues: []string{
```

Comment:
no API is available to fetch this list ? 

Should we ask for a new endpoint ?

Reply:
We could do it with a call on ListOffer. I suggest we make it a different PR.